### ISSUE TITLE
bug(gatsby-plugin-fastify): using deprecated packages

### ISSUE THREAD

**Issue by tsdexter** [gatsby-uc/plugins#223]
## Describe the bug

I can't build due to deprecation errors: 

`HttpErrorGeneral: 404 Not Found - GET https://registry.npmjs.org/fastify-accepts-deprecated - Not found` and several other related errors with other fastify packages

It looks like all the fastify packages were moved to the `@fastify` scope

## To Reproduce

Do a new install  

## Expected behavior

No errors

## System Info

Please provide information about your site via these means as possible:

  System:
    OS: macOS 12.2
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 16.13.1 - /var/folders/jf/5ss8k9vn7v91sn37s8b8zgzc0000gn/T/yarn--1654178606018-0.1492902467831616/node
    Yarn: 1.22.4 - /var/folders/jf/5ss8k9vn7v91sn37s8b8zgzc0000gn/T/yarn--1654178606018-0.1492902467831616/yarn
    npm: 8.1.2 - ~/.nvm/versions/node/v16.13.1/bin/npm
  Languages:
    Python: 2.7.18 - /usr/bin/python
  Browsers:
    Chrome: 102.0.5005.61
    Firefox: 99.0.1
    Safari: 15.3
  npmPackages:
    gatsby: ^4.13.1 => 4.13.1 
    gatsby-plugin-fastify: ^0.8.1 => 0.8.1 
    gatsby-plugin-gatsby-cloud: ^4.13.0 => 4.13.0 
    gatsby-plugin-image: ^2.13.0 => 2.13.0 
    gatsby-plugin-manifest: ^4.13.0 => 4.13.0 
    gatsby-plugin-offline: ^5.13.0 => 5.13.0 
    gatsby-plugin-postcss: ^5.3.0 => 5.3.0 
    gatsby-plugin-react-helmet: ^5.13.0 => 5.13.0 
    gatsby-plugin-sharp: ^4.13.0 => 4.13.0 
    gatsby-source-filesystem: ^4.13.0 => 4.15.0 
    gatsby-source-wordpress: ^6.15.0 => 6.15.0 
    gatsby-transformer-sharp: ^4.13.0 => 4.13.0

**Comment by moonmeister** [comment]
Hmmm, I know those fastify packages were moved with the release of fastify@v4. We haven't made any updates that should break things that I'm aware of. But I've been out of touch. Let me check some things.

**Comment by moonmeister** [comment]
Yeah, looks like the package still exists under the old name and should be available to install from npm. I'm not sure why it'd be 404ing. It should work even if depreciated.

**Comment by moonmeister** [comment]
Yeah, looks like the package still exists under the old name and should be available to install from npm. I'm not sure why it'd be 404ing. It should work even if depreciated.

**Comment by tsdexter** [comment]
@moonmeister thx for the update from afar... I thought it was strange also... not sure if NPM was having a bad day or something else was causing it, but it was only happening on WP Engine Atlas platform, not locally. It looks like it's gotten passed that when rebuilding today and is not complaining about `stack Error: not found: make` ... I'll come back and close this if appropriate after a bit more testing
